src/reducers/reducer.js

```diff
@@ -2131,3 +2132,8 @@ export const onUnregisteredTokensDownloadEnd = (state) => ({
     isLoading: false,
   },
 });
+
+export const onSetNetworkName = (state, action) => ({
+  ...state,
+  networkName: action.payload,
```

Comment:
Shouldn't we have the default value for `networkName`? Even if it's `null` or empty string, at least we could have it there with a docstring.

Reply:
Yes! Fixed, thanks